### PR TITLE
Add ZWNJ separator between emojis to not join them when decoding

### PR DESCRIPTION
**Pull request by 0x6e756c6c**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,28 +26,48 @@
...
     },
-    decrypt (str = '💩') {
-      const startIndex = table.indexOf(str.slice(0, 2))
-      const base = str.toString().slice(2, str.toString().length).replace(/../g, e => {
+    decrypt (str = '💩', debug = false) {
+      const emojis = str.split(ZWNJ)
+      const start = emojis.shift()
+      const startIndex = table.indexOf(start)
+
+      if (debug) {
```
Comment (wvffle) [inline index.js:41]: I think that all of that debug stuff is useless ;/

**Review by wvffle — CHANGES_REQUESTED** [review]
Remove the debug stuff

**Comment by 0x6e756c6c** [comment]
Well it's actually kind of useful to see where the error is. Take a look at this function:
```js
    _test () {
      let pass = true
      for (let i = 0; i < 10000; ++i) {
      	const str = Math.random() + ''
        const enc = emojicrypt.encrypt(str)
        try {
          const dec = emojicrypt.decrypt(enc)
          if (dec != str) {
            throw new Error('src/dec mismatch: ' + str + ' / ' + dec)
          }
        } catch (e) {
          console.error(i + ': ' + enc)
          console.error(e)
          try {
            emojicrypt.decrypt(enc, true)
          } catch (e) {}
          pass = false
          break;
        }
      }

      return pass
    }
```

That helped me to find that you were assuming that emojis are always 2 characters long what was breaking the decoder if using joined emojis. Or as in the screenshot that first I was counting starting emoji to the base64 string:
image

**Comment by wvffle** [comment]
Okay, so add that method to the emojicrypt object.

**Review by wvffle — APPROVED** [review]

**Pull request merged**